identity/manager.go

```diff
@@ -270,3 +297,26 @@ func (m *Manager) CountActiveMultiFactorCredentials(ctx context.Context, i *Iden
 	}
 	return count, nil
 }
+
+// Fandom-start https://github.com/Wikia/kratos/pull/84
+func (m *Manager) GetIdentityHighestAAL(ctx context.Context, identityId uuid.UUID) (available AuthenticatorAssuranceLevel, err error) {
```

Comment:
Nitpick: This code is used only in `selfservice/strategy/link/strategy_recovery.go` (after applying Suggestion [3] from `manager_http`). We could move this code into `strategy_recovery`

Reply:
I moved this code to `strategy_recovery`.